File: PlusTrack/PlusTrack.API/Domain/Entities/RouteStop.cs
namespace PlusTrack.API.Domain.Entities
{
    public class RouteStop
    {
        public Guid Id { get; set; }
        public int StopOrder { get; set; }

        public Route? Route { get; set; }
        public Guid? RouteId { get; set; }

        public Location? Location { get; set; }
        public Guid? LocationId { get; set; }

        public Package? Package { get; set; }
        public Guid? PackageId { get; set; }
    }
}

File: Plustrack_android/app/src/main/java/com/example/plustrack_android/RouteStop.kt
package com.example.plustrack_android

import java.io.Serializable
import java.util.UUID

data class RouteStop(
    val id: String,
    val stopOrder: Int,
    val location: Location? = null,
    val locationId: String? = null,
    val pack: Package? = null,
    val packageId: UUID? = null
) : Serializable
